db/migrate/20190805140757_create_image_table.rb

```diff
@@ -1,4 +1,6 @@
 class CreateImageTable < ActiveRecord::Migration[5.2]
+  XML_HEADER = "<?xml version=\"1.0\"?>".freeze
```

Comment:
If this is part of a service we should be able to reference it as opposed to re creating it.

Reply:
Oops, yeah I could have definitely done that. Updated the reference.